dev-docs/auth.md

```diff
@@ -15,15 +15,28 @@ We added our own [authn-service](https://github.com/chef/authn-service) as glue
 
 For further questions, the team can be reached at [#auth-team](https://chefio.slack.com/messages/C61F9HHKK/)
 
-## OpenID Connect aka Dex
+## Supported Identity Management Systems
 
-If you want to use google as an IdP (Identity Provider) or our internal test LDAP server, there are a few variables you can set.
+OKTA with SAML and AD with LDAP have anecdotally been reported to be the most common configurations among our users.
+We have also had success with the following configurations:
 
-To use a test OIDC provider we set up, look for `test oidc provider` on lastpass. Export the value for `Username` as `GOOGLE_CLIENT_ID` and the value for `Password` as `GOOGLE_CLIENT_SECRET`.
+### SAML
 
-For our test ldap server, look up `ldap testhost` and set `LDAP_HOST` to the `Hostname` value. Also look up `ldapbind user` and
-set `LDAP_BIND_DN` and `LDAP_BIND_PW` to the values for `Username` and `Password`, respectively.
+- OKTA
+- Ping
+- OneLogin
+- Office365
+- Tivoli Federated Identity Manager
 
+### LDAP
+
+- Microsoft Active Directory (MSAD)
+
+### Configurations with Known Issues
```

Comment:
lets move this section to somewhere in the repo instead of in the docs

Reply:
these are dev docs so they're not part of the docs we deploy to customers. but i could move it to the authn README if that seems a little more out of the way

Reply:
oh lol sorry, ignore me.